etc/gz.bash_completion.sh

```diff
@@ -0,0 +1,35 @@
+# gz bash completion
```

Comment:
I'm forward-porting the bash completion in #95, let's get that one in first, and then we can move the file here.

Reply:
#95 is in

Reply:
Ported!

Reply:
I left the `ign.bash_completion.sh` alone for now though